### PR TITLE
[Behat] Checked whether a customer receives the welcoming email

### PR DESCRIPTION
**Pull request by NeverResponse**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Related tickets | - 
| License         | MIT

### CODE REVIEW

File: src/Sylius/Behat/Context/Ui/Shop/RegistrationContext.php [hunk 1]
```diff
@@ -406,6 +406,25 @@ public function numberOfEmailsShouldBeSentTo($count, $email)
     }
 
     /**
+     * @Then a welcoming email should have been sent
+     */
+    public function aWelcomingEmailHaveBeenSent()
+    {
+        $this->emailContainsMessage('Welcome to our store');
```
Comment (pjedrzejewski) [inline src/Sylius/Behat/Context/Ui/Shop/RegistrationContext.php]: assertEmailContainsMessage

Comment (lchrusciel) [inline src/Sylius/Behat/Context/Ui/Shop/RegistrationContext.php]: Do we even need this extraction?

File: src/Sylius/Behat/Context/Ui/Shop/RegistrationContext.php [hunk 2]
```diff
@@ -406,6 +406,25 @@ public function numberOfEmailsShouldBeSentTo($count, $email)
     }
 
     /**
+     * @Then a welcoming email should have been sent
+     */
+    public function aWelcomingEmailHaveBeenSent()
```
Comment (lchrusciel) [inline src/Sylius/Behat/Context/Ui/Shop/RegistrationContext.php]: aWelcomingEmail **Should** HaveBeenSent

File: features/account/registration.feature [hunk 3]
```diff
@@ -47,3 +47,10 @@ Feature: Account registration
         Then I should be notified that new account has been successfully created
         And I should be logged in
         And my email should be "goodman@gmail.com"
+
+    @ui @email
+    Scenario: Receiving a welcoming email after registration
+        When I register with email "ghastly@bespoke.com" and password "suitsarelife"
+        Then I should be notified that new account has been successfully created
+        And a welcoming email should have been sent
```
Comment (lchrusciel) [inline features/account/registration.feature]: It just begging to be merged with the step below.
```Gherkin
And a welcoming email should have been sent to "ghastly@bespoke.com"
```

**Comment by michalmarcinkowski** [comment]
Thanks Janek!

**Pull request merged**